### PR TITLE
Feat - 163073639-ui-fixes

### PR DESCRIPTION
**Pull request by xcixor**
## Purpose
Fix some ui issues
## Tasks completed
-Add border to breadcrumb
-Remove borders from form inputs
-Add means to add apple icon to db
-Add phone-categories links in missing pages
## PT story 
[#163073639](https://www.pivotaltracker.com/story/show/163073639)
### Border to breadcrumb
screenshot from 2019-01-11 18-50-51
### Apple icon
screenshot from 2019-01-11 18-51-14
### Remove form input borders
screenshot from 2019-01-11 18-51-58
### Add links to phone categories such as in news and press page
screenshot from 2019-01-11 19-03-54

### CODE REVIEW

File: hirola/front/views.py [hunk 1]
```diff
@@ -104,9 +104,11 @@ def phone_profile_view(request, phone_id):
     colors = PhonesColor.objects.filter(phone=phone_id, is_in_stock=True)
     if not phone:
         return redirect("/error")
+    (phone_categories, social_media) = various_caches()
```
Comment (JamesKirkAndSpock) [inline hirola/front/views.py]: For all the view methods, also pass the social_media variable just as you passed the phone_categories variable.😔I forgot to include it.

File: hirola/front/views.py [hunk 2]
```diff
@@ -386,7 +400,9 @@ def search_view(request):
                         phone_color_quantity__is_in_stock=True,
                         phone_color_quantity__quantity__gte=1))
         results = list(set(results))
-        args = {"results": results, "instructions": False}
+        args = {"results": results, "instructions": False,
```
Comment (JamesKirkAndSpock) [inline hirola/front/views.py]: What is the purpose fo the instructions argument being passed?

File: hirola/front/templates/front/landing_page.html [hunk 3]
```diff
@@ -73,9 +73,4 @@ <h5>Hot deals!</h5>
 {% block footer %}
 {% include 'front/footer.html' %}
 {% endblock %}
-{% block scripts %}
-{{ block.super }}
-{% load static %}
-<script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
 {% endblock %}
```
Comment (JamesKirkAndSpock) [inline hirola/front/templates/front/landing_page.html]: Add a line here

File: hirola/front/views.py (hunk 2 above)
Comment (xcixor) [reply hirola/front/views.py]: I honesttly dont know, you are the one who put it, should I remove it

**Pull request merged**